lib/assessment.md

```diff
@@ -21,28 +21,46 @@ Using this format will make it easier for us to grade you, so that we can get ev
 ## Question 1
 
 When we say that the web is a service, what does that mean? Explain the interaction in terms of clients and servers.
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+The Web provides a service to access information over the Internet. It is built on top of the Internet.
 
 ## Question 2
 
 Between the comments below, write HTML to create a basic web page (using the format described in the instructions). The page should have a title in the head and an unordered list (with three items) inside the body.
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+<!DOCTYPE html>
+  <html>
+    <head>
+    <!-- Meta-data goes here. -->
+    <title>html Diagnostic</title>
+    </head>
+    <body>
+    <ul>
+    <li>One</li>
+    <li>Two</li>
+    <li>Three</li>
+    </ul>
+      <!-- Page content goes here. -->
+    </body>
+  </html>
 
 ## Question 3
 
 What selectors could we write to reference each of the following categories of elements?
 * All elements that belong to the class `big`
+  .big{
+  }
 * The element matching ID `contentPane`
+  #contentPane{
+
+  }
 * Any elements that belong both to the `important` and `red` classes
+  .important, .red{
+
+  }
```

Comment:
The wording on this prompt was a little ambiguous, so I'll give you the points.
We actually wanted only those elements that belonged to **both** classes (i.e. the **intersection**). The selector you've given will select all members of the `important` class AND all members of the `red` class, including but not limited to elements belonging to both classes; this is the **union** of the two classes.

The selector to grab only elements belonging to both classes would be `.important.red`